lib/sync-UpdatesModel_mongo.js

```diff
@@ -1,19 +1,9 @@
-var syncUtil = require('./sync-util');
-var MongoClient = require('mongodb').MongoClient;
 var fhdb;
 var mongo;
 
 var setFHDB = function(db) {
   fhdb = db;
-};
-
-var setConnectionUrl = function(url) {
-  MongoClient.connect(url, function(err, db) {
-    if (err) throw err;
-    syncUtil.doLog(syncUtil.SYNC_LOGGER, 'info', 'sync-UpdatesModel MongoClient connected');
-
-    mongo = db;
-  });
+  mongo = fhdb.getMongoClient();
```

Comment:
This is giving an error
```js
/home/dmartin/work/fh-mbaas-api/lib/sync-UpdatesModel_mongo.js:6
  mongo = fhdb.getMongoClient();
               ^

TypeError: fhdb.getMongoClient is not a function
```
I think the function may not be exported completed through fh-db and to `db.js` in fh-mbaas-api.
i'll take a look into it.
I'm using master fh-db, npm linked into this fh-mbaas-api.